packages/disputer/src/disputer.js

```diff
@@ -94,7 +94,9 @@ class Disputer {
 
     // Get the latest disputable liquidations from the client.
     const undisputedLiquidations = this.empClient.getUndisputedLiquidations();
-    const disputeableLiquidations = undisputedLiquidations.filter(liquidation => {
+    // Push disputable liquidations along with their dispute prices to this array.
+    let disputableLiquidationsWithPrices = [];
+    undisputedLiquidations.map(liquidation => {
```

Comment:
If you wait on #2493, this loop becomes a map anyway, so you could just return the liquidation object + a price without having to construct the separate array.

Reply:
Would you just modify the loop to be `return { ...liquidation, price}` in the successful outcome?

Reply:
Yeah, basically.